style.css

```diff
@@ -0,0 +1,204 @@
+* {
+background-color: #e9f0ea;
+}
+
+body {
+text-align: center;
+font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+}
+
+header {
+position: absolute;
+top: 0;
+right: 0;
+padding: 10px 18px;
+}
+
+.header-box {
+display: flex;
+flex-direction: row-reverse;
+}
+
+.sign-in-txt {
+margin-right: 8px;
+display: inline;
+}
+
+#sign-in-btn {
+background-color: #0384fc;
+border: #000000;
+border-radius: 4px;
+text-align: center;
+cursor: pointer;
+padding: 11px 24px;
+color: azure;
+font-size: 14px;
+}
+
+#member {
+margin-right: 8px;
+display: inline;
+}
+
+.icons-trey {
+text-align: center;
+}
+
+.form-box{
+width: 345px;
+background-color: #ffffff;
+border: 2px solid #e2dede;
+margin: auto;
+text-align: center;
+height: 100%;
+padding-top: 2px;
+padding-bottom: 30px;
+margin-top: 30px;
+margin-right: 150;
+margin-right: 150;
```

Comment:
Don't forget to add units for your margin values, otherwise, they are deemed invalid by the browser.